sound/soc/sof/ipc4-topology.c

```diff
@@ -1976,12 +1978,30 @@ static int sof_ipc4_control_setup(struct snd_sof_dev *sdev, struct snd_sof_contr
 	return 0;
 }
 
+static int sof_ipc4_pipeline_send_kcps(struct snd_sof_dev *sdev, struct snd_sof_widget *swidget,
+				       __s32 adjust)
```

Comment:
what does 'adjust' mean? is this to specify a kcycles value, or alter it from a baseline read in topology?

Completely unclear what this is about.

Reply:
I can call it "delta" if that would help

Reply:
or `kcps_change`?

Reply:
if you call it 'delta' then the function should be not sof_ipc4_pipeline_send_kcps(), sof_ipc4_send_kcps_update()